test/integration/targets/get_certificate/files/process_certs.py

```diff
@@ -0,0 +1,25 @@
+from sys import argv
+from subprocess import Popen, PIPE, STDOUT
+
+p = Popen(["openssl", "s_client", "-host", argv[1], "-port", "443", "-prexit", "-showcerts"], stdin=PIPE, stdout=PIPE, stderr=STDOUT)
+stdout = p.communicate(input=b'\n')[0]
+data = stdout.decode()
```

Comment:
Could PyOpenSSL/cryptography be used here for data parsing rather than doing it manually ? `module_utils/crypto.py` or other crypto modules have example on how to be used.

Reply:
I think this isn't parsing the certificate, but extracting the raw PEM file from the output generated by openssl's integrated s_client.